File: corpus/2018q1/_p2q2_ok/_check/C-;0375.py
static int n=25
static int next=0
String nome = ""
int opcao=0
print("--Entre com as informações. Digite 0 para sai. ")
print("--Digite 1 para Cadastrar")
print("--Digite 2 para Pesquisar ")
while opcao!=3:
    opcao=int(input())
    if opcao==1:
        Nome(Nome,Destino)
    else:
        pesquisarDestino(Nome, Destino,nome)
    print("--Digite 0 para sair ou apenas continue cadastrando. ")
    opcao=int(input())
    next=next +1
def Nome(String Nome [],String Destino[]):
    print("Informe o nome do passageiro: ")
    print("Informe o destino: ")
def pesquisarDestino(String Nome [],String Destino[], String nome):
    print("Informe o nome a ser localizado: ")
    nome = entry.nextLine()
    for =0 in range(0, n):
        print("Nome:"+Nome[i]+"")
        print("Destino:"+Destino[i]+"")
        else:
            print("Null")

File: corpus/2018q1/_p2q2_ok/_check/C-;0375.java

import java.util.Scanner;

public class Q2_ {
static int n=25;
static int next=0;

    public static void main(String[] args) {
        Scanner ent = new Scanner(System.in);
        String Nome[]= new String[n];
        String Destino[] = new String[n];
        String nome = "";
        int opcao=0;

        System.out.println("--Entre com as informações. Digite 0 para sai. ");
        System.out.println("--Digite 1 para Cadastrar");
        System.out.println("--Digite 2 para Pesquisar ");

        while(opcao!=3){
        opcao=ent.nextInt();
        if (opcao==1){
        Nome(Nome,Destino);
        }
        else if (opcao==2){
        pesquisarDestino(Nome, Destino,nome);
        }
        System.out.println("--Digite 0 para sair ou apenas continue cadastrando. ");
        opcao=ent.nextInt();             
        next=next+1;

        }

    }
    public static void Nome(String Nome [],String Destino[]){
        Scanner ent = new Scanner(System.in);

            System.out.println("Informe o nome do passageiro: ");
            Nome[next]=ent.nextLine();
            System.out.println("Informe o destino: ");
            Destino[next]=ent.nextLine();

    }
    public static void pesquisarDestino(String Nome [],String Destino[], String nome){
    Scanner entry = new Scanner(System.in);
    System.out.println("Informe o nome a ser localizado: ");
    nome = entry.nextLine();

        for(int i =0; i<n;i++){
           if(nome==Nome[i]){
        System.out.println("Nome:"+Nome[i]+"");
        System.out.println("Destino:"+Destino[i]+"");
        }
           else{
        System.out.println("Null");
        }

        }
}
}
